ansible/roles/collectory/tasks/vm-tasks.yml

```diff
@@ -18,21 +18,75 @@
     - db
     - collectory
 
+
+#deploy as systemd service
+- name: add collectory jar and setup systemd service
+  include_role:
+    name: exec-jar
+  vars:
+    service_name: "{{collectory_app | default('collectory')}}"
+    use_openjdk17: "{{ collectory_version | string is version('5.2.0', '>') }}"
+    log_config_filename: "{{ logback_config_filename }}"
+    java_headless: True
+  tags:
+    - deploy
+    - service
+    - collectory
+  when: exec_jar | default(false) | bool
+
+- name: Extract assets (images) from jar for systemd service
```

Comment:
If `Extract assets (images) from jar for systemd service` needs to be removed. This is only required if the collectory jar is not built correct. Fix it in the collectory build process.